cli/tests/common/mod.rs

```diff
@@ -1,10 +1,13 @@
+use std::env;
 use std::ffi::OsStr;
+use std::fs::{self, File};
+use std::io::Write;
 use std::path::{Path, PathBuf};
-use std::{env, fs};
 
 pub use assert_cmd::assert::Assert;
 pub use assert_cmd::Command;
 use phylum_cli::api::{PhylumApi, PhylumApiError, ResponseError};
+use phylum_cli::commands::extensions::permissions::Permissions;
```

Comment:
I'm not sure whether it would be better to condition the whole module on `#[cfg(feature = "extensions")]` or to put this only where applicable. We aren't really running integration tests that don't have this feature yet, but might in the future, unless we plan on removing the feature eventually.

Reply:
I've just added a bunch of attributes, should hopefully be fine until the feature is published.